src/core/positioning/nmeagnssreceiver.h

```diff
@@ -51,10 +51,38 @@ class NmeaGnssReceiver : public AbstractGnssReceiver
     void handleStartLogging() override;
     void handleStopLogging() override;
 
+    void handleIMUCorrection( const QString &sentence );
```

Comment:
Just to match convention set in QgsNmeaConnection class, rename to processImuSentence would be nice.